### DISCUSSION TITLE
New v0.2.0 version

### DISCUSSION THREAD

**Discussion by 4rtzel** [4rtzel/poe-archnemesis-scanner#47] [Announcements]
* Add a "Shopping List" feature that allows you to specify the recipes that you're interested in. Only these recipes, and the recipes that are needed to create them, will show up in scanning results (thanks https://github.com/williammetcalf).
* Add a Scan/Hide hotkeys.
* Add an option to disable overlay mode.

<hr /><em>This discussion was created from the release <a href='https://github.com/4rtzel/poe-archnemesis-scanner/releases/tag/v0.2.0'>New v0.2.0 version</a>.</em>

**Comment by JustLKS** [comment]
I am sorry if this is a retarded question... but how to use the "shopping" feature? I have no idea what I need to do to set my favorite recipes!! I mean, should I edit some file or something? Ty!!

**Reply by 4rtzel** [reply]
Have you checked the README? There is a description on how to use those features (check the Settings section).

Please let me know if it's still not clear from the README how to use it, because I'll need to improve the documentation in that case.

**Reply by JustLKS** [reply]
Thank you I'll take a look at it. Btw: the tool now after update is showing juggernaut as frenzied. Some wrong detection on this is happening. Thanks!

**Reply by JustLKS** [reply]
> Have you checked the README? There is a description on how to use those features (check the Settings section).
> 
> Please let me know if it's still not clear from the README how to use it, because I'll need to improve the documentation in that case.

I am sorry sir, but I am still confused about shopping list.
How am I supposed to do it?
I mean, I want kitava touched recipe (for example).
How to place it on .ini file???
Thank you!!!

**Reply by 4rtzel** [reply]
You don't place it in the .ini. You do it in the settings next to 'Set shopping list' button and then press the button. The name of the recipe should be correct and with right letter case. For kitava it would be 'Kitava-Touched'.

**Comment by maxtraxv3** [comment]
says cant detect poe screen, when i have file manger on full screen, when its in a smaller screen, seem to detect it, but the interface button disappear as soon as i click on poe...

EDIT: seem i now have to use window default file manger, for it to work.

**Reply by 4rtzel** [reply]
I don't understand. Could you elaborate? How file manager has to do with anything?

**Reply by maxtraxv3** [reply]
im using this file system instead of the windows 11 default "https://files.community" last verion ran fine when using it but now the interface doesn't work unless i use the default windows file manger.

**Reply by 4rtzel** [reply]
That's quite strange. The file managers shouldn't do anything fancy by themselves when launching executables. Maybe there is a problem with relative path?

**Comment by masterbalans** [comment]
'Display unavailable recipes' not work.  When choosing next to 'Used in:' recipe show in full? ver 0.1.0 all recipe show full monitor///

**Reply by 4rtzel** [reply]
Are you using 'Shopping list' feature? It will filter out all recipes not in the list.

**Reply by 4rtzel** [reply]
Also, check that your settings are the same because they will set to default when you download a new version.

**Reply by masterbalans** [reply]
yes thank you very much!!! the list of recipes was included and empty (()), changed the settings and everything works
